src/pages/MyReportsHistory/components/Content/Content.tsx

```diff
@@ -10,7 +10,7 @@ const Content: FC = () => {
   const [viewMode, SetViewMode] = useState<'list' | 'card'>('list')
 
   return (
-    <div className="w-[975px] bg-[#E9EFF4] pt-14 px-20">
+    <div className="w-[975px] bg-[#E9EFF4] pt-14 px-20 h-full">
```

Comment:
I might prefer min-h-screen here as h-full depends on the parent container height, what if the height changed or maybe not set explicitly

Reply:
actually I think there might be another way, always set the children(right hand side of the FlexLayout component) be min-h-screen, and then remove the min-h-screen on the Container in FlexLayout component, which would be better?

Reply:
please ignore first two suggestions as they still not working properly for my reports page, I tried add another flex in FlexLayout component as below:
```suggestion
    <div className="flex-grow flex">{children}</div>
```
funny this actually fixed my reports page, and you can remove h-full

Reply:
and if you can help to add "bg-white" for signup and help-support(might need adjust width or paddings) page, and remvoe h-[832px] for login page, that should make all pages consistent

Reply:
> 

哈哈哈，确实，好奇怪啊